python/Makefile

```
@@ -85,21 +85,21 @@ pkg-test-%: packages/%
 	true'
 else
 pkg-test-%: packages/%
-	UV_CACHE_DIR="$(CURDIR)/.uv-cache/$*" uv run --isolated --directory $< pytest || [ $$? -eq 5 ]
+	uv run --isolated --directory $< pytest || [ $$? -eq 5 ]
 endif
 
 pkg-ty-%: packages/%
 	uv run --isolated --directory $< ty check .
 
-pkg-test-all: $(addprefix pkg-test-,$(PKG_TARGETS))
+pkg-test-all: sync $(addprefix pkg-test-,$(PKG_TARGETS))
```

checkmake (0.3.2)

[warning] Target "pkg-test-all" should be declared PHONY.

(phonydeclared)